opensrp-anc/src/main/java/org/smartregister/anc/library/sync/BaseAncClientProcessorForJava.java

```diff
@@ -244,6 +258,40 @@ public void updateFTSsearch(String tableName, String entityId, ContentValues con
         return new String[]{DBConstantsUtils.KeyUtils.ANC_ID, ConstantsUtils.JsonFormKeyUtils.ANC_ID};
     }
 
+    @Override
+    public void processEventClient(@NonNull EventClient eventClient, @NonNull List<Event> unsyncEvents, @Nullable ClientClassification clientClassification) throws Exception {
+        Event event = eventClient.getEvent();
+        if (event == null) {
+            return;
+        }
+
+        String eventType = event.getEventType();
+        if (eventType == null) {
+            return;
+        }
+
+        switch (eventType) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=4809639)